src/pages/Dashboard.vue

```diff
@@ -51,9 +51,10 @@ export default defineComponent({
     }
     let unsubscribe: VoidFn = () => {}
     let clientData: ClientData = emptyData
-    return { lang, config, network, plot, global: global.data, globalState, expanded: false, util, loading: true, unsubscribe, clientData }
+    return { lang, config, network, plot, global: global.data, client: global.client, globalState, expanded: false, util, loading: true, unsubscribe, clientData }
   },
   async mounted() {
+    this.client.init()
```

Comment:
Add TODO to remove this once invariants are protected. Someone WILL use `this.client` at some point when they forget to call `this.client.init()` in another component and things WILL break in a racy way, I have seen enough applications that are broken like this.